python_modules/libraries/dagster-azure/dagster_azure/adls2/__init__.py

```diff
@@ -14,7 +14,6 @@
 )
 from .resources import (
     ADLS2DefaultAzureCredential as ADLS2DefaultAzureCredential,
-    ADLS2FileManager as ADLS2FileManager,
```

Comment:
?

Reply:
This is deleted because `ADLS2FileManager` is already exported above.